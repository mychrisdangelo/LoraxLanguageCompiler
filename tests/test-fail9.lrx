/* Tests tree type mismatch*/

int main () {
	
	tree <int> t (3);
	t = 1[2, 5[3, 5, 4.0], 5];
	
}