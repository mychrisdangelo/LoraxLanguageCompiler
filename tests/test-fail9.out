Fatal error: exception Failure("Tree literal type is not consistent: expected <int> but received <float>")
